### PR TITLE
Standardize colors across reporters.

### PR DESCRIPTION
**Pull request by jamestalmage**
PR #410 missed a color change on a single log line,
See: https://github.com/sindresorhus/ava/pull/410#issuecomment-170210415

Using this convention will make it much easier to review PR's. No need to remember what color is used where, just ensure they don't use chalk directly, and alias the color with it's intended purpose in the colors module.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -21,9 +21,9 @@ var updateNotifier = require('update-notifier');
 var figures = require('figures');
 var arrify = require('arrify');
 var meow = require('meow');
-var chalk = require('chalk');
 var Promise = require('bluebird');
 var pkgConf = require('pkg-conf');
+var colors = require('./lib/reporters/colors');
```
Comment (vdemedes) [inline cli.js]: `lib/reporters` was intended to contain files of reporters only. `lib/colors.js` would be just fine as well

Comment (jamestalmage) [inline cli.js]: fixed

**Comment by sindresorhus** [comment]
LGTM

**Comment by kevva** [comment]
:+1:

**Pull request closed without merging**